test/unit/server_tests.rb

```diff
@@ -172,9 +121,9 @@ class InitTests < UnitTests
       @error_procs = Factory.integer(3).times.map{ proc{} }
       @error_procs.each{ |p| @server_class.error(&p) }
 
-      @server_class.router do
-        service Factory.string, TestHandler.to_s
-      end
+      # @server_class.router do
+      #   service Factory.string, TestHandler.to_s
+      # end
```

Comment:
@jcredding no it's not.  This was supporting the detailed router validation tests.  I've moved these to the router tests since I abstracted that logic on the router.  I should have deleted this.  Here is where I do the validation tests in the router tests: https://github.com/redding/sanford/pull/173/files#diff-3e16e407c4d571d506ac8150f25bdf81R65.